xbatcher/loaders/keras.py

```diff
@@ -1,6 +1,12 @@
 from typing import Any, Callable, Optional, Tuple
 
-import tensorflow as tf
+try:
+    import tensorflow as tf
+except ImportError:
+    raise ImportError(
+        "The Xbatcher TensorFlow Dataset API depends on TensorFlow. Please "
+        "install TensorFlow to proceed."
+    )
```

Comment:
Raise from original exception??

```suggestion
except ImportError as exc:
    raise ImportError(
        "The Xbatcher TensorFlow Dataset API depends on TensorFlow. Please "
        "install TensorFlow to proceed."
    ) from exc
```

Reply:
Good catch, thanks!